cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetDataset.java

```diff
@@ -118,7 +118,7 @@ protected void addPartitionToExplore(PartitionKey key, String path) {
       ExploreFacade exploreFacade = exploreFacadeProvider.get();
       if (exploreFacade != null) {
         try {
-          exploreFacade.addPartition(getName(), key, files.getLocation(path).toURI().getPath());
+//          exploreFacade.addPartition(getName(), key, files.getLocation(path).toURI().getPath());
```

Comment:
Not dead code. This is currently blocked by some changes needed in PFS for explore to work.